samples/experimental/skills-buffered/parent/app.py

```diff
@@ -12,11 +12,11 @@
     MemoryStorage,

```

Comment:
Not sure if it is worth standardizing on the folders where we put these end to end tests, in C# we put them under FunctionalTests/Skills. Ignore if things in the python repo are organized differently.

Reply:
Axel and I talked about this.  In R9/10 we'll do some cleanup.  There are other places where there are test bots, and it's inconsistent.  But... C# also has test bots scattered about.